### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by mikedropmedia**
Updated hero copy to address Rodrigo's and John's comments

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@ <h1>
             Your go-to online machine learning library
           </h1>
           <p class="description">
-            Vowpal Wabbit is a fast and flexible learning system that empowers you to test and explore real data and find meaningful outcomes to real problems.
+            Vowpal Wabbit is a fast and flexible online learning system that empowers meanigful interactions to help you solve complex learning problems.
```
Comment (kumpera) [inline index.html]: The new wording is inaccurate and confusing.

File: index.html [hunk 2]
```diff
@@ -94,7 +95,7 @@ <h2>
             A vehicle for machine learning research
           </h2>
           <p class="description">
-            Vowpal Wabbit’s design and capabilities are informed and influenced by advanced research and proven algorithms. Researchers who wish to implement their algorithm in Vowpal Wabbit have access to quick deployment and extensive use.
+            Vowpal Wabbit’s design is informed and influenced by advanced research and proven algorithms to empower online learning research capabilities.
```
Comment (kumpera) [inline index.html]: The last part of the edit is wrong.

**Comment by cheng-tan** [comment]
this PR needs to open against "source" branch. Deleted "homepag" branch and opening the same PR with "source"

File: index.html (hunk 1 above)
Comment (mikedropmedia) [reply index.html]: @JohnLangford  thoughts? We can ditch meaningful interactions: "Vowpal Wabbit is a fast and flexible online learning system for complex learning problems." I now see "real world" used across other Microsoft Research content, so if we need to return to that fine. As long as it is the most accurate, most specific hook that encourages further exploration and use of VW.

File: index.html (hunk 2 above)
Comment (mikedropmedia) [reply index.html]: What would you suggest instead? @JohnLangford My take here was we want to highlight the research and science behind VW. If the original "quick deployment and extensive use" copy that I used here is not accurate how can we improve the copy? Vehicle, power, and drive work nicely in this section so I'd like to iterate with that them intact.

**Pull request closed without merging**